_posts/2024-08-14-diving-into-jtag-part-6.md

```diff
@@ -194,21 +194,21 @@ transfer data from input to output, minimizing latency.
 1. The first step is similar to the first step in the previous method.
 2. Send the sequence `0b01100` on the selected `TMS` pin to move thus to the
    `Shift-IR` state.
-3. Now it is necessary to send the `BYPASS` command, but since the size of the
-   `IR` register is not known, but it is necessary to send a large number of
-   units (as many as 1024, for example).
+3. Now, it is not necessary to send the `BYPASS` command, but since the size of the
+   `IR` register is not known, it is necessary to send a large number of
+ units (as many as 1024, for example).
```

Comment:
@Zamuhrishka Could you confirm that this is the correct wording? I inferred from the language

Reply:
Not quite. You don't need to use 'not' here because we need to use the `BYPASS` command.
And  I noticed that I missed the part about the "units", which needs to be corrected. It would be better to write 'ones' instead of 'units.'. My fault...